TradeDis/frontend
# Navigation Between Homepage and Create Post

**What was done?**
Issue: TRAD-81 "Navigate to Create Post page from Home Feed"

- Navigate to Create Post page by clicking "plus" (+) button on the home feed
- Navigate back from Create Post page to Home Feed by clicking on the "back" button
- Navigate back from Create Post page to Home Feed after successful post creation

**Areas which need special reviewing attention**

- Possibly the navigation implementation in `/navigation/index.ts` in the `RootNavigator` function. I saw many examples online that used this layout to navigate to different screens and I think we should continue with this format.

**Testing**
Navigation from Home Feed to Create Post
1. Ensure both the backend server (`yarn dev` in the backend project) and the frontend (`expo start` in the frontend project) are running
2. On the simulator, go to the Home Feed screen
3. Click on the "plus" (+) location at the bottom of the screen
4. You should be navigated to the Create Post screen

Navigation from Create Post to Home Feed
1. Ensure both the backend server (`yarn dev` in the backend project) and the frontend (`expo start` in the frontend project) are running
2. On the simulator, go to the Home Feed screen and then the Create Post screen
3. Once on the Create Post screen, click on the "back" button located on the top left
4. You should be navigated to the Home Feed screen

Navigation from Create Post to Home Feed
1. Ensure both the backend server (`yarn dev` in the backend project) and the frontend (`expo start` in the frontend project) are running
2. On the simulator, go to the Home Feed screen and then the Create Post screen
3. Once on the Create Post screen, create a post and press the "Post!" button
4. You should see a success message under the "Post!" button and should be redirected to the Home Page screen
5. Additionally, you should see your newly added posts in both carousels
